recipes/evo/recipe.yaml

```diff
@@ -0,0 +1,66 @@
+context:
+  version: 1.31.1
+  python_min: 3.10
+
+package:
+  name: evo
+  version: ${{ version }}
+
+source:
+- url: https://files.pythonhosted.org/packages/37/c9/450f89b92ad6fccf66152267dfb52f6bf9e85476b217a14ac4a09a5ccadd/evo-${{ version }}.tar.gz
```

Comment:
Please use the URL as in the example: https://github.com/conda-forge/staged-recipes/blob/main/recipes/example-v1/recipe.yaml#L23

This one will break the bot updates.